src/firefly/js/core/background/BackgroundMonitor.jsx

```diff
@@ -264,23 +256,19 @@ function PackageItem({SINGLE, jobId, index}) {
 
 
     const action = dlreq?.WS_DEST_PATH
-        ? (<Typography justifyContent={'flex-end'} width={175} height={20} sx={{color:'dimgray', mx:'0.5', my:'auto', display: 'inline-flex',
-            whiteSpace: 'nowrap',
-        }}>
+        ? (<Link>
```

Comment:
and @robyww in my cleanup I noticed there was never an actual link for `Download to Workspace` (so if `dlreq?.WS_DEST_PATH` exists, then you just see the text for `Download to Workspace`). Left this as is for now.